pkgs/development/python-modules/lm-format-enforcer/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pydantic,
+  interegular,
+  pyyaml,
+  poetry-core
+}:
+
+buildPythonPackage rec {
+  pname = "lm-format-enforcer";
+  version = "0.10.4";
+  format = "pyproject";
```

Comment:
i can never remember... is the latest guidance `pyproject = true;` or ` format = "pyproject";`?

Reply:
`pyproject = true`